netlify.toml

```diff
@@ -12,6 +12,55 @@
     max-age=31536000;
     includeSubDomains;'''
 
+    Content-Security-Policy = '''
+    frame-ancestors
+      'self';
+    upgrade-insecure-requests
+      1;
+    script-src
+      'self'
+      'report-sample'
+      'unsafe-inline'
+      eval:
+      *;
+    style-src
+      'self'
+      'report-sample'
+      'unsafe-inline'
+      *;
+    object-src
+      'none';
+    frame-src
+      'self'
+      *;
+    child-src
+      'self';
+    img-src
+      'self'
+      data:
+      *;
+    font-src
+      'self'
+      *.cloudfront.net;
```

Comment:
should we include a * here ?

Reply:
this CloudFront address it's from netlify (I think) so, for some reason, the subdomain could change? 🤔

Reply:
sorry, I meant font-src * self *.cloudfront.net

Reply:
oh, IDK. When I was checking, the only domain that load fonts was this CloudFront 🤔